### PR TITLE
LSR-21087: Translate Work Order Template to Supported Languages

### PR DESCRIPTION
**Pull request by ls-lindsay-powerharding**
Add workorder templates for all supported languages

Note: Translations are in a google doc attached to the Jira

### CODE REVIEW

File: workorder/customizable_workorder_de.tpl [hunk 1]
```diff
@@ -0,0 +1,550 @@
...
+Set any of the options in this section from 'false' to 'true' in order to enable them in the template
+#}
+
+{% set chrome_right_margin_fix = false %}			{# Fixes a potential issue where the right side of receipts are cut off in Chrome #}
+{% set firefox_margin_fix = false %}				{# Fixes issue with margins cutting off when printing on a letter printer on a Mac #}
+
+{% set itemized_hours_labor = false %}				{# Shows time spent on Labor item if time spent is present in the charge #}
+{% set per_line_subtotal = false %}					{# Displays Subtotals for each Sale Line (ex. 1 x $5.00) #}
+{% set employee_name_on_labor_charges = false %}	{# Display Employee name on Labor Charges #}
+{% set make_work_order_number_small = false %}		{# Makes work order number inline with the header #}
```
Comment (tombertrand) [inline workorder/customizable_workorder_de.tpl:12]: Nit: Could you align these attribute descriptions?

File: workorder/customizable_workorder_de.tpl [hunk 2]
```diff
@@ -0,0 +1,550 @@
...
+				<td data-automation="lineItemQuantity" class="quantity">{{Line.unitQuantity}}</td>
+			{% endif %}
+		{% else %}
+			<td data-automation="lineItemQuantity" class="quantity">{{Line.unitQuantity}}</td>
+		{% endif %}
+		<td data-automation="lineItemRowCharge" class="amount">
+			{% if Line.warranty == 'false' %}
+			    {{Line.SaleLine.displayableSubtotal|money}}
+			{% elseif Line.warranty == 'true' %}
+			    $0.00
```
Comment (tombertrand) [inline workorder/customizable_workorder_de.tpl:519]: might be euro symbol for `de`?

File: workorder/customizable_workorder_nl.tpl [hunk 3]
```diff
@@ -0,0 +1,550 @@
...
+				<td data-automation="lineItemQuantity" class="quantity">{{Line.unitQuantity}}</td>
+			{% endif %}
+		{% else %}
+			<td data-automation="lineItemQuantity" class="quantity">{{Line.unitQuantity}}</td>
+		{% endif %}
+		<td data-automation="lineItemRowCharge" class="amount">
+			{% if Line.warranty == 'false' %}
+			    {{Line.SaleLine.displayableSubtotal|money}}
+			{% elseif Line.warranty == 'true' %}
+			    $0.00
```
Comment (tombertrand) [inline workorder/customizable_workorder_nl.tpl:519]: might be euro `€` symbol for `nl`?

**Review by tombertrand — APPROVED** [review]
🌮  w/ nits

**Review by brandon-dev-aleriola — APPROVED** [review]

**Review by ls-deepak-yadav — APPROVED** [review]

File: workorder/customizable_workorder_de.tpl (hunk 1 above)
Comment (ls-lindsay-powerharding) [reply workorder/customizable_workorder_de.tpl:12]: Sure thing!

**Pull request merged**